spec/support/pages/full_work_package_create.rb

```diff
@@ -27,6 +27,7 @@
 #++
 
 require 'support/pages/page'
+require 'support/pages/abstract_work_package_create'
```

Comment:
This was required to make the tests run on my machine, otherwise I'd get the same error as in https://github.com/opf/openproject-ce/pull/6#issuecomment-194311860